frontend/utils/feedback.py

```diff
@@ -170,8 +171,15 @@ def show_feedback_form(
     feedback = st.sidebar.text_area("Please provide your feedback or report an issue:")
 
     if selected_question:
-        sources = [metadata[selected_question].get("sources", "N/A")]
-        context = [metadata[selected_question].get("context", "N/A")]
+        sources = metadata[selected_question].get("sources", ["N/A"])
```

Comment:
Can you explain this? Is it supposed to be a list of lists?

Reply:
so the `submit_feedback_to_google_sheet` function was returning an error even when the input was right, apparently what was causing this issue was the source and context was being passed in as a `str` but the function accepted a list, so I changed it to have a type-safety that whenever an independent string is being passed, it is changed to a list of single string so the function works